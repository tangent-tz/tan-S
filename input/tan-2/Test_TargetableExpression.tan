main {
    var a := new [int](5); 
    print a \n; 
    
    [a:1] := 120; 
    print a \n; 
    
    var b:= 'a'; 
    print b \n; 
    
    b := 'b'; 
    print b \n; 
    
    
    [1+1 : 3-2] := true; 
        

}