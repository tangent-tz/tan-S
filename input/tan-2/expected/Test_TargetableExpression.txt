[ 0, 0, 0, 0, 0 ] 
[ 0, 120, 0, 0, 0 ] 
[ 0, 120, 0, 0, 1 ] 
[ 0, 120, 345, 0, 1 ] 
a
b
h
[ hi, hello ] 
[ what, hello ] 
[ i, t ] 
[ w, t ] 
[ 1.200000, 4.644000 ] 
[ 546.640000, 4.644000 ] 
[ false, true ] 
[ true, true ] 
